src/VerifyWebauthnController.js

```diff
@@ -197,23 +197,14 @@ export default FormController.extend({
       return children;
     },
 
-    postRender: function () {
-      _.defer(() => {
-        if (webauthn.isNewApiAvailable()) {
-          this.model.save();
```

Comment:
who makes this `save` call now? Or we didn't need it before? 
Oh wait... or is it that now we don't move the UI automatically to CHALLENGE, but instead now the user clicks 'verify' button, which submits the form and makes a `model.save` request, which is what replaces this specific call. In summary we don't make this call automatically for a user, which to me makes sense

Reply:
yeah and this is what Safari expects and allows.